multipart/Cargo.toml

```diff
@@ -1,17 +1,18 @@
 [package]
-name = "multipart-example"
-version = "0.1.0"
-authors = ["Nikolay Kim <fafhrd91@gmail.com>"]
-workspace = ".."
+name = "actix-web-upload-example"
+version = "0.0.2"
+authors = ["Bevan Hunt <bevan@bevanhunt.com>"]
 edition = "2018"
-
-[[bin]]
-name = "multipart"
-path = "src/main.rs"
+license = "MIT"
+description = "Simple file uploader in Actix Web with Async/Await"
+keywords = ["actix", "actix-web", "multipart"]
+repository = "https://github.com/bevanhunt/actix-web-upload-example"
+readme = "README.md"
 
 [dependencies]
-actix-web = "1.0.0"
-actix-multipart = "0.1.1"
-
-env_logger = "0.6"
-futures = "0.1.25"
+futures = "0.3.1"
+actix-multipart = "0.2.0-alpha.1"
+actix-web = "2.0.0-alpha.1"
+envy = "0.4.0"
```

Comment:
Why `envy` instead of `env_logger`? What do the other examples use?

Reply:
`env_logger` I will change that - thanks.

Reply:
Removed the need for env_logger/envy as hard-coded port.